models/calendar.yml

```diff
@@ -288,10 +288,10 @@
 
     - 9:00 PM:
         type: Talk
-        title: TBD
-        emoji: 🤔
+        title: Push and Pull: Boids!
```

Comment:
Because of the `:`, the parser is getting confused. If you write it like so, it should work:

```yml
title: |
  Push and Pull: Boids!
```